control_msgs/action/FollowJointTrajectory.action

```diff
@@ -50,3 +54,7 @@ string[] joint_names
 trajectory_msgs/JointTrajectoryPoint desired
 trajectory_msgs/JointTrajectoryPoint actual
 trajectory_msgs/JointTrajectoryPoint error
+
+string[] multi_dof_joint_names
+trajectory_msgs/MultiDOFJointTrajectoryPoint multi_dof_desired
+trajectory_msgs/MultiDOFJointTrajectoryPoint multi_dof_actual
```

Comment:
Seems like there should be a --

`trajectory_msgs/JointTrajectoryPoint error`

to match L56

Reply:
Its redundant, but might as well.